recipes-kernel/linux/linux-toradex/SPI-CAN.cfg

```diff
@@ -0,0 +1,14 @@
+CONFIG_SPI=y
```

Comment:
This is a lot of config elements and might overlap with other config stuff in weird ways. Do we really need all of this? Can we pare it down to a minimum configuration set that doesn't overlap with everything else? Or was there really absolutely no spi support built into our kernel before this?